app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::UsersController < ApplicationController
+  skip_before_action :authenticate_user!, :only => [:show]
+
+  def show
```

Comment:
Wondering if there's any risk to this. Basically, it allows the user to get their information (user id, name) if they're logged in. Could change it to just return the name, which is all we need.